src/frontend/src/utils/statusList/statuses/editInProgressStatus.ts

```diff
@@ -52,6 +53,7 @@ export const editInProgressStatus = () => {
       },
       CPO: {
         showSubmitButton: true,
+        showEditButton: true,
```

Comment:
Shouldn't this be under Pages?

Reply:
> Shouldn't this be under Pages?

yes I think so too I have changed under Pages!